imap-proto/src/parser/rfc5256.rs

```diff
@@ -0,0 +1,27 @@
+//!
+//! https://tools.ietf.org/html/rfc5256
+//!
+//! SORT extension
+//!
+
+use nom::{
+    bytes::streaming::{tag, tag_no_case},
+    combinator::{map, opt},
+    multi::many0,
+    sequence::{preceded, terminated},
+    IResult,
+};
+
+use crate::{parser::core::number, types::MailboxDatum};
+
+pub(crate) fn mailbox_data_sort(i: &[u8]) -> IResult<&[u8], MailboxDatum> {
+    map(
+        // Technically, trailing whitespace is not allowed here, but multiple
+        // email servers in the wild seem to have it anyway (see #34, #108).
```

Comment:
Have you actually seen this for this particular extension?

Reply:
In fact the `SORT` command is very close to the `SEARCH` command (it kind of extends it). So I guess they share the same "issues". Should I remove?

Reply:
I guess it's fine to keep, but may make it explicit that this is conjecture based on the similarity to `SEARCH`?